### PR TITLE
Geomap: update the layer order in docs

### PR DESCRIPTION
**Pull request by ryantxu**
This updates the TOC so the layer types are ordered more reasonably:

| Before | After |
|----|---|
| image | image |

### CODE REVIEW

**Comment by ryantxu** [comment]
I'm not sure if this needs a backport or not

File: docs/sources/visualizations/geomap/daynight.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ keywords:
   - panel
   - documentation
 title: Night / Day layer
-weight: 400
+weight: 6
```
Comment (drew08t) [inline docs/sources/visualizations/geomap/daynight.md:12]: Should this be the same as OSM?

Comment (ryantxu) [reply docs/sources/visualizations/geomap/daynight.md]: updated

**Review by drew08t — APPROVED** [review]

**Comment by JoaoSilvaGrafana** [comment]
Hey @ryantxu the backport failed on this PR, did you want to do a manual backport before the 9.1.1 release tomorrow?

**Pull request merged**